### ISSUE TITLE
Floating mode in bspwm causes high CPU usage

### ISSUE THREAD

**Issue by Vintodrimmer** [brndnmtthws/conky#425]
Good day.

I'm running the default config except for `own_window_type = 'normal'` and `font = 'Fira Mono:size=13'`.

If I launch the Conky in the tiling mode, everything is normal (90% of the time at least), but if I make the window floating or even simply move the tile, the cpu load skyrockets to 20-22% and stays there. Launching in the floating mode yields the same high load.

Here's the `conky -v`:

```
conky 1.10.6_pre compiled Sat Dec 10 05:22:59 UTC 2016 for Linux 4.2.5_1 x86_64

Compiled in features:

System config file: /etc/conky/conky.conf
Package library path: /usr/lib/conky


 General:
  * math
  * hddtemp
  * portmon
  * IPv6
  * Curl
  * RSS
  * Weather (METAR)
  * Weather (XOAP)
  * wireless
  * support for IBM/Lenovo notebooks
  * builtin default configuration
  * old configuration syntax
  * Imlib2
  * apcupsd
  * iostats
  * ncurses
  * Internationalization support

 Lua bindings:
  * Cairo
  * Imlib2
  * RSVG
 X11:
  * Xdamage extension
  * Xinerama extension (virtual display)
  * Xshape extension (click through)
  * XDBE (double buffer extension)
  * Xft
  * ARGB visual
  * Own window

 Music detection:
  * MPD
  * MOC

 Default values:
  * Netdevice: eth0
  * Local configfile: $HOME/.conkyrc
  * Localedir: /usr/share/locale
  * Maximum netdevices: 64
  * Maximum text size: 16384
  * Size text buffer: 256
```

Distribution is Void Linux with musl if that helps.

What may be the problem here?

![Conky started in tiling mode](https://raw.githubusercontent.com/Vintodrimmer/docs/master/2017-10-26-18%3A56%3A24-screenshot.png)
![Conky moved in tiling mode](https://raw.githubusercontent.com/Vintodrimmer/docs/master/2017-10-26-18%3A56%3A28-screenshot.png)
![Conky started in floating mode](https://raw.githubusercontent.com/Vintodrimmer/docs/master/2017-10-26-18%3A59%3A42-screenshot.png)

**Comment by easysid** [comment]
I can confirm this issue with bspwm on Arch linux.

[Floating mode causes high cpu usage](https://i.imgur.com/aPyQwjh.png?1). Not sure if it is a bspwm issue or a conky one.

**Comment by lasers** [comment]
10 months passed. Can you determine if you're still having this problem today on `1.10.9` or preferably `1.10.10_pre` (git)? The older versions are not trustworthy due to too many changes that can be hard to track. Thank you.

**Comment by Vintodrimmer** [comment]
I'll check, but isn't the latest release still `1.10.8`?

**Comment by Vintodrimmer** [comment]
Thanks for the detailed instructions, but I'm still curious about the version 1.10.9. There seems to be no corresponding tag on GitHub or am I missing something?

And yes, unfortunately, the bug is still there for me.
Here's a free floating "default" window:
![Default window](https://raw.githubusercontent.com/Vintodrimmer/docs/master/2018-08-12-09%3A48%3A09-screenshot.png)
And fixed "dock" variant:
![Dock](https://raw.githubusercontent.com/Vintodrimmer/docs/master/2018-08-12-09%3A48%3A22-screenshot.png)

**Comment by plikhari** [comment]
Yes the current released version is 1.10.8 - git under development version is 1.10.9_pre

**Comment by lasers** [comment]
`bswpm` with and without `conky`.

![2018-09-06-083717_900x601_scrot copy](https://user-images.githubusercontent.com/852504/45166112-e42daf00-b1bb-11e8-9889-b94b654c19cb.png)
